packages/core/src/core.ts

```diff
@@ -8,6 +8,6 @@ core.register([
   coreNodeProvider,
 ]);
 
-core.bootSync();
+core.boot();
```

Comment:
It might make more sense to export this _unbooted_, ie leaving it for the developer to boot it, instead of returning `{ core: Promise<Application> }`?